docs/pages/hdmi-cec-sink-L3-Low-Level_TestSpec.md

```diff
@@ -0,0 +1,346 @@
+    RAFT/user->>DUT: Start the HAL test
+    DUT->>DUT: Add Logical Address
+    DUT->>DUT: Set receive Callback
+    DUT->>CEC Adaptor: Broadcast "standby"
+    RAFT/user->>CEC Adaptor: Validate the CEC Command received on the `CEC` adaptor
+    DUT->>DUT: Wait for the input from RAFT/user to provide the test result
+    RAFT/user->>DUT: Provide the Test result of standby data received by it.
+    DUT->>DUT: Wait for the input from RAFT/user to send a broadcast command
+    RAFT/user->>DUT: RAFT/user to signal the test to move ahead
+    RAFT->>CEC Adaptor: Broadcast "standby" command
+    RAFT/user->>DUT: Signal the command to validate the data RAFT/user broadcasted and received by DUT.
+    CEC Adaptor->> DUT: Broadcast the standby command
+    DUT-->>DUT: Validate the received command
+```
+
+#### Test Procedure 
+
+| Variation / Steps | Description | Test Data | Expected Result | Notes |
+|-------------------|-------------|-----------|------------------|-------|
+| 01 | Open HDMI CEC HAL using `HdmiCecOpen` API | `handle` = valid pointer | `HDMI_CEC_IO_SUCCESS` | Should be successful |
+| 02 | Acquire a valid logical address `0x00` using `HdmiCecAddLogicalAddress` | `handle` = valid handle, `logicalAddress` = 0 | `HDMI_CEC_IO_SUCCESS` | Should be successful |
+| 03 | Set the receive callback function using `HdmiCecSetRxCallback` | `handle` = valid handle, `cbfunc` = RxCallback, `data` = buffer pointer | `HDMI_CEC_IO_SUCCESS` | Should be successful |
+| 04 | Frame a standby command to broadcast from `DUT` to all connected devices using `HdmiCecTx`. Test command and data shall be read from the test YAML file | `handle` = valid handle, `buf` = standby command as read from YAML file, `len` = sizeof(buf), `result` = valid pointer | `HDMI_CEC_IO_SUCCESS` | Should be successful |
+| 05 | RAFT/user should receive this data and validate the command received. The test should wait for the RAFT to enter the test result based on the validation | N/A | N/A | RAFT to provide the result |
+| 06 | RAFT/user to command CEC adaptor to frame and broadcast the standby CEC command. Test data should be read from the test YAML file | `buffer` = test YAML file data | N/A | RAFT to set this data through CEC adaptor |
+| 07 | Wait for the RAFT to signal when the CEC broadcast is sent in step 6 | RAFT/user to provide `y` to move the test | N/A | RAFT to signal once the CEC broadcast is set |
+| 08 | Validate the data received from the CEC adaptor on the test RxCallback against the test YAML file | Test data received should be as mentioned in the test YAML file | Read and validate this data | Should be successful |
+| 09 | Close HDMI CEC HAL using `HdmiCecClose` API | `handle` = valid handle | `HDMI_CEC_IO_SUCCESS` | Should be successful |
+
+# Test 3: Stress Test
+
+Functionality: 
+1. `DUT` shall respond to a CEC OSD command received from the CEC Adaptor repeatedly for 10 times with different strings and validate.
```

Comment:
not enough information to figure out what should happen here.

Where's the yaml message that will work for both the vDevice & the platform code.

And the CEC adaptor is adnostic to the testing suite.